README.md

```diff
@@ -265,8 +265,8 @@ Code blocks can also be added:
       | B { 'B' }
       ;
 
-When the terminal `A` would be processed the returned value would be "B", for
-terminal `B` the returned value would be "B".
+When the terminal `A` would be processed the returned value would be `'A'`, for
+terminal `B` the returned value would be `'B'`.
```

Comment:
Oh wait that's right, they're quoted in the example blocks. In that case this is good to go.